Components/Spinner/stories.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { doc } from 'storybook-readme';
+import README from './README.md';
+import Spinner from "./index";
+
+storiesOf('Spinner', module).add('Overview', doc(README));
+storiesOf("Spinner", module).addWithChapters('Display',  {
+  chapters: [
+    {
+      sections: [
+        {
+          title: "Standard",
+          subtitle: "Default set to 'loading'",
+          info: "Indicates a neutral informative change or action.",
+          sectionFn: () => (
+            <Spinner title="Hello">Button text</Spinner>
+          )
+        }, {
```

Comment:
Is it worth including examples in our design guide for things we can't see (i.e. aren't being rendered)? Would we have an example for a button which isn't visible?

If it was a different visual state (e.g. the spinner is greyed out or something), then this would totally make sense. But I don't see the value.

Reply:
Good point. Hmmmm... I'll remove the `enabled` story, as it's explained (at least touched on) in the Overview